### ISSUE TITLE
change of lib directory brake haxelib

### ISSUE THREAD

**Issue by ibilon** [HaxeFoundation/haxelib#191]
Doing another `haxelib setup` and then moving the lib folder will break haxelib since the path to the "real" haxelib is hardcoded.

```
#!/bin/sh
OLDCWD=`pwd`
cd /hardcoded/path/to/haxelib/haxelib_client/3,2,0-rc,3/src/
exec haxe -D haxelib_client=3.2.0-rc.3 --run tools.haxelib.Main -cwd $OLDCWD $@
```

changing the cd to
```
cd $(cat ~/.haxelib)/haxelib_client/3,2,0-rc,3/src/
```
would prevent that.

**Comment by back2dos** [comment]
That might do the trick, although you cannot really assume that `~/.haxelib` is the right place to look for the path. 

@jasononeil Maybe we could use something more basic, like the path where haxe itself located?

**Comment by ibilon** [comment]
@back2dos The error happened to me because I moved the lib folder away from where haxe is (so I can switch haxe version more easily)

**Comment by nadako** [comment]
"fixed" by #291
